Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1782,28 +1798,23 @@ void CGContextDrawGlyphRun(CGContextRef context, const DWRITE_GLYPH_RUN* glyphRu
 }
 #pragma endregion
 
-CGImageRef CGPNGImageCreateFromFile(NSString* path) {
-    return new CGPNGDecoderImage([path UTF8String]);
-}
-
-CGImageRef CGPNGImageCreateFromData(NSData* data) {
-    return new CGPNGDecoderImage(data);
-}
-
-CGImageRef CGJPEGImageCreateFromFile(NSString* path) {
-    return new CGJPEGDecoderImage([path UTF8String]);
-}
-
-CGImageRef CGJPEGImageCreateFromData(NSData* data) {
-    return new CGJPEGDecoderImage(data);
-}
-
 #pragma region CGBitmapContext
 struct __CGBitmapContextImpl {
     woc::unique_cf<CGImageRef> image;
 };
 
-struct __CGBitmapContext : CoreFoundation::CppBase<__CGBitmapContext, __CGBitmapContextImpl, __CGContext> {};
+struct __CGBitmapContext : CoreFoundation::CppBase<__CGBitmapContext, __CGBitmapContextImpl, __CGContext> {
+    inline ComPtr<ID2D1RenderTarget>& RenderTarget() {
```

Comment:
>  inline ComPtr<ID2D1RenderTarget>& RenderTarget() { [](start = 2, length = 52)

We can't do this, unfortunately. It's not virtual, so callers who think they're calling CGContext::RenderTarget will be statically compiled to refer to it (and it may be inlined.)

I think this logic belongs in `CGBitmapContextCreate`, or a helper, which will allow us to back out this change, and the other `SetRenderTarget` change.